Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -305,6 +305,29 @@ public AugenPriceSpike APS(Symbol symbol, int period = 3, Resolution? resolution
             return bollingerBands;
         }
 
+        /// <summary>
+        /// Creates a Alpha indicator for the given target symbol in relation with the reference used.
+        /// The indicator will be automatically updated on the given resolution.
+        /// </summary>
+        /// <param name="target">The target symbol whose Alpha value we want</param>
+        /// <param name="reference">The reference symbol to compare with the target symbol</param>
+        /// <param name="alphaPeriod">The period of the Alpha indicator</param>
+        /// <param name="betaPeriod">The period of the Beta indicator</param>
+        /// <param name="resolution">The resolution</param>
+        /// <param name="riskFreeRate">The risk free rate</param>
+        /// <param name="selector">Selects a value from the BaseData to send into the indicator, if null defaults to casting the input value to a TradeBar</param>
+        /// <returns>The Alpha indicator for the given parameters</returns>
+        [DocumentationAttribute(Indicators)]
+        public Alpha A(Symbol target, Symbol reference, int alphaPeriod = 1, int betaPeriod = 252, Resolution? resolution = null, decimal riskFreeRate = 0.0m, Func<IBaseData, IBaseDataBar> selector = null)
```

Comment:
Could you take a look at https://github.com/QuantConnect/Lean/pull/7594 and see how the `SharpeRaitio` indicator takes a nullable risk free ratio and decides whether to use the algorithm's risk free rate model or a constant rate? So you can make this indicator follow the same pattern. See both the `SR()` method and the `SharpeRatio` class constructors.